causal_testing/testing/estimators.py

```diff
@@ -306,9 +306,6 @@ def __init__(
             terms = [treatment] + sorted(list(adjustment_set)) + sorted(list(effect_modifiers))
             self.formula = f"{outcome} ~ {'+'.join(terms)}"
 
-        for term in self.effect_modifiers:
```

Comment:
This still needs to be done if no formula is provided, or is this take account of elsewhere?

Reply:
I wanted to ask about these lines! I wasn't sure where/if the adjustment set was used after this. 

None of the unittests seem to cover this being used either so I'll raise that as an issue and re-add the lines :).

Reply:
Yes, effect modifiers need to be added to the adjustment set. Causally, they are semantically different to normal adjustment variables but, at least for LR, they are treated the same way when you do the maths.